### DISCUSSION TITLE
Discussions are now enabled

### DISCUSSION THREAD

**Discussion by emabrey** [tenacityteam/tenacity-legacy#101] [Announcements]
At the request of many Github users, @emabrey has enabled discussions. 

### **Be respectful and follow the rules of GitHub at all times.**

If abuse becomes a problem we unfortunately may need to disable discussions. If that happens there may not be a good location for non-issue public comments until we complete the chatroom process. So _please_ act respectfully.

**Comment by WeedyWeedSmoker** [comment]
The topic for name suggestions is closed, but I think the best name would start similarly to Audacity…

**AudacityOTB**
(_Out of The Box, Of This Bitch… You decide the meaning for OTB!_ 😂)
